### ISSUE TITLE
I want to integrate GroupDocs Viewer for NET in my company program

### ISSUE THREAD

**Issue by Faayakhun** [groupdocs-free-consulting/projects#46]
I want to inquire about GroupDocs License, if I use GroupDocs library in my NET program with License (assuming its only Viewer License for Developer Small Business), will my program still works to render/view document stored locally when there is no internet connection?, or it needs continuous internet connection in order for it to works?, thanks in advance.

**Comment by atirtahirgroupdocs** [comment]
@Faayakhun 

> will my program still works to render/view document stored locally when there is no internet connection?

Yes, application will work.  You can load documents from different sources (e.g. local path string). Please take a look at this documentation [article](https://docs.groupdocs.com/viewer/net/loading-documents-from-different-sources/).
